.github/workflows/ci.yml

```diff
@@ -0,0 +1,110 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      OCI_REGISTRY: ghcr.io
+      DOCKER_IMAGE_NAME: yieldlab/prebid-server-java
+
+    steps:
+    - name: Checkout version file
+      uses: actions/checkout@v3
+      with:
+        sparse-checkout: |
+          latest-release-version
+        sparse-checkout-cone-mode: false
+
+    - name: Extract version
+      run: |
+        target_tag=$(cat latest-release-version)
+        echo "TARGET_TAG=$target_tag" >> $GITHUB_ENV
+
+    - name: Check version already exist
+      run: |
+        ghcr_token=$(curl https://$OCI_REGISTRY/token\?scope\="repository:$DOCKER_IMAGE_NAME:pull" | jq -r '.token')
+        status_code=$(curl --silent \
+                  -H "Authorization: Bearer $ghcr_token" \
+                  -H "Accept: application/vnd.docker.distribution.manifest.v2+json" \
+                  -o /dev/null -L -w "%{response_code}" \
+                  https://$OCI_REGISTRY/v2/$DOCKER_IMAGE_NAME/manifests/$TARGET_TAG)
+        [[ "$status_code" -ne 404 ]] && tag_existing="true" || tag_existing="false"
+        echo "TAG_ALREADY_EXISTING=$tag_existing" >> $GITHUB_ENV
+
+    - name: Checkout prebid-server-java
+      uses: actions/checkout@v3
+      with:
+        repository: 'prebid/prebid-server-java'
+        ref: ${{ env.TARGET_TAG }}
+
+    - name: Set up JDK 17
+      uses: actions/setup-java@v3
+      with:
+        distribution: corretto
+        java-version: 17
+
+    - name: Check environment
```

Comment:
I think we could leave out this step. I don't think we should print out the environment as it will be publicly visible. Also `jq` is already being used before this step.